### PR TITLE
Support bionic clients.

### PR DESCRIPTION
**Pull request by alienth**
:eyeglasses: @spladug @rram

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+hyperlink==18.0.0
+idna==2.8
+incremental==17.5.0
+ipaddress==1.0.22
+pyOpenSSL==19.0.0
+pyasn1==0.4.5
+pyasn1-modules==0.2.5
+pycparser==2.19
+pycrypto==2.6.1
+rollingpin==1.0
```
Comment (spladug) [inline requirements.txt]: should probably drop this line -- hard to install itself!

Comment (alienth) [reply requirements.txt]: haha, ty

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Automat==0.6.0
+Twisted==17.5.0
+ZooKeeper==0.4
+argparse==1.2.1
+asn1crypto==1.0.1
+attrs==19.1.0
+cffi==1.10.0
+constantly==15.1.0
```
Comment (rram) [inline requirements.txt]: I'm just excited about multi-line comments

File: rollingpin/transports/ssh.py [hunk 3]
```diff
@@ -64,8 +64,7 @@ def getPublicKey(self):
             return succeed(public_key)
 
     def getPrivateKey(self):
-        key = self.factory.key.keyObject
-        return succeed(key)
+        return succeed(self.factory.key)
```
Comment (rram) [inline rollingpin/transports/ssh.py]: 🔕 Does this work in old twisted as well? Perhaps all of your commits should be squashed into one.

**Pull request merged**